SideMenu.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "SideMenu"
-  s.version          = "4.0.0"
+  s.version          = "4.0.1"
```

Comment:
I guess a new major should be introduced? Since it's technically a breaking change on the version if Swift 4.2 isnt supported

Reply:
Should jump to `5.0.0`